scripts/dice.coffee

```diff
@@ -0,0 +1,66 @@
+# Description:
+#   Allows Hubot to roll dice
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   hubot roll (die|one) - Roll one six-sided dice
+#   hubot roll dice - Roll two six-sided dice
+#   hubot roll <x>d<y> - roll x dice, each of which has y sides
+#   hubot roll <x>dF - roll x fudge dice, each of which has +, +, 0, 0, -, and - sides.
+#
+# Author:
+#   ab9
+#   https://github.com/github/hubot-scripts/blob/master/src/scripts/dice.coffee
```

Comment:
If its in hubot-scripts, can we just add this to https://github.com/desert-planet/arrakis-hubot/blob/master/hubot-scripts.json?